backends/qualcomm/quantizer/quantizer.py

```diff
@@ -22,25 +22,29 @@
 from torch.ao.quantization.quantizer import Quantizer
 from torch.fx import GraphModule
 
-from .utils import (
+from .qconfig import (
+    get_16a16w_qnn_ptq_config,
     get_16a4w_qnn_ptq_config,
+    get_16a4w_qnn_qat_config,
     get_16a8w_qnn_ptq_config,
-    get_default_16bit_qnn_ptq_config,
-    get_default_8bit_qat_proto,
-    get_default_8bit_qnn_ptq_config,
+    get_8a8w_qnn_ptq_config,
+    get_8a8w_qnn_qat_config,
```

Comment:
As a follow-up PR, can we unify/simplify: `get_8a8w_qnn_ptq_config()` and `get_8a8w_qnn_qat_config()` into `get_8a8w_qnn_config()` with the argument is_train=True/False, which will define whether we are doing PTQ vs. QAT.

Reply:
No problem. We would like to keep separated at first. Once it seems to be stable we will raise a TODO PR.